src/common/definitions.h

```diff
@@ -22,6 +22,26 @@
 #define DONT_OPTIMIZE // silently ignore on Visual Studio, where this is less of a problem
 #endif
 
+// Use these macros to enable faster math.  Put them around one or more
+// functions.  Usage:
+//
```

Comment:
Could you add to the comment what the trade-offs are? I.e. what are we (potentially) losing by switching to faster math and why is that OK?

Reply:
Long comment. 3078927